Library/Homebrew/cmd/readall.rb

```diff
@@ -13,16 +13,8 @@ module Homebrew
 
   def readall
     if ARGV.include?("--syntax")
-      ruby_files = []
-      scan_files = %W[
-        #{HOMEBREW_LIBRARY}/*.rb
-        #{HOMEBREW_LIBRARY}/Homebrew/**/*.rb
-      ]
-      Dir.glob(scan_files).each do |rb|
-        next if rb.include?("/vendor/")
-        next if rb.include?("/cask/")
-        ruby_files << rb
-      end
+      scan_files = "#{HOMEBREW_LIBRARY}/Homebrew/**/*.rb"
```

Comment:
This doesn't match what it says above as it won't cover `#{HOMEBREW_LIBRARY}/*.rb`

Reply:
Sorry, missed @reitermarkus's comment above; ignore this.

Reply:
Still, `#{HOMEBREW_LIBRARY}/Homebrew` should be replaced with `HOMEBREW_LIBRARY_PATH`.

Reply:
Fixed.